content/features/content-management/index.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: features-content-management
```

Comment:
We are using the ID convention without the "feature-" in front of the item in this section as it is part "feature list" and part "glossary". I guess "features-general" can stay, but this one should be "content-management" to stay in line with the rest of the content.

Reply:
@donker if we don't differentiate between content in this Features section (which is the reason for the "features-" prefix) and content in the Tutorials sections, we are going to run into **uid** conflicts for sure.  I was honestly unaware of (or perhaps just forgot) any specific **uid** convention that has previously been established.  How would you like to proceed?